appservice/src/tree/DeploymentTreeItem.ts

```diff
@@ -105,6 +105,13 @@ export class DeploymentTreeItem extends AzureTreeItem<ISiteTreeRoot> {
                 await this.parent.refresh(); /* refresh entire node because active statuses has changed */
                 window.showInformationMessage(redeployed);
                 ext.outputChannel.appendLine(redeployed);
+            } catch (error) {
+                const parsedError: IParsedError = parseError(error);
+                if (parsedError.message.includes('Repository could not be found')) {
```

Comment:
This would cover most cases, but I don't like checking for specific text in an error message because it won't work if/when the error gets translated.

Can you do a smarter check before you even try to redeploy? Is there a property on the deployment that says how it was created? Would it make sense to just do `siteConfig.scmType === ScmType.None` again?

Reply:
Nah, a user can redeploy to a git commit even if their ScmType is None.  I'll investigate the DeployResult objects.

Reply:
Think we could use this more reliably?

Reply:
isReadonly sounds good as long as you've tested it

Reply:
Pssssht, of course, I tested it...more than once...